package.json

```diff
@@ -133,5 +133,5 @@
     "singleQuote": true,
     "trailingComma": "es5"
   },
-  "stardog-version": ">=7.0.0"
+  "stardog-version": ">=8.1.0"
```

Comment:
i think this is fine as `>=8.0.0`

Reply:
you won't be able to use this new endpoint on 8.0.0 though; is that ok?

Reply:
yep that's ok as long as it's not breaking any existing features in 8+

Reply:
[`c0086a5` (#276)](https://github.com/stardog-union/stardog.js/pull/276/commits/c0086a5175d6b4b91c04059cec00e1216bb2b0c7)